src/test/jtx/TestHelpers.h

```diff
@@ -461,6 +461,100 @@ create(
 
 }  // namespace check
 
+static constexpr FeeLevel64 baseFeeLevel{256};
+static constexpr FeeLevel64 minEscalationFeeLevel = baseFeeLevel * 500;
+
+template <class Suite>
+void
+checkMetrics(
+    Suite& test,
+    int line,
+    jtx::Env& env,
+    std::size_t expectedCount,
+    std::optional<std::size_t> expectedMaxCount,
+    std::size_t expectedInLedger,
+    std::size_t expectedPerLedger,
+    std::uint64_t expectedMinFeeLevel = baseFeeLevel.fee(),
+    std::uint64_t expectedMedFeeLevel = minEscalationFeeLevel.fee())
+{
+    FeeLevel64 const expectedMin{expectedMinFeeLevel};
+    FeeLevel64 const expectedMed{expectedMedFeeLevel};
+    auto const metrics = env.app().getTxQ().getMetrics(*env.current());
+    using namespace std::string_literals;
+
+    metrics.referenceFeeLevel == baseFeeLevel
+        ? test.pass()
+        : test.fail(
+              "reference: "s +
+                  std::to_string(metrics.referenceFeeLevel.value()) + "/" +
+                  std::to_string(baseFeeLevel.value()),
+              __FILE__,
```

Comment:
`__FILE__` is no longer going to be accurate - it'll point to this file

Reply:
fixed: https://github.com/XRPLF/rippled/pull/5060/commits/9e85f3c290ef67302fd51ff3de7b32bf850b0343